Tatsi/Config/TatsiConfig.swift

```diff
@@ -87,6 +90,9 @@ public struct TatsiConfig {
             LocalizableStrings.tableName = localizableStringsTableName
         }
     }
+    
+    /// Set this to a number of PHAsset instances that were selected before the picker was opened.
+    public var preselectedAssets: [PHAsset]? = nil
```

Comment:
my suggestion is to declare this as an empty array like `public var preselectedAssets = [PHAsset]()` because that is the actual default behaviour, that none are preselected. nil as default does not have an extra meaning here over an empty array, I think.